src/config/environment/index.js

```diff
@@ -0,0 +1,25 @@
+/**
+ * @fileoverview Config file to centralize environmental variables read
+ *
+ * This file is intended to centralize access to environmental variables,
```

Comment:
Livvie recommended writing comments starting with a verb. I know a wrote this, so I can rewrite it if you want.

Reply:
Yes, np!